secant/Util/RecoveryPhraseStorage.swift

```diff
@@ -220,7 +220,7 @@ extension RecoveryPhraseStorage: KeyStoring {
         }
     }
     
-    func areKeysPresent() throws -> Bool {
+    func areKeysPresent() -> Bool {
```

Comment:
⚠️ Warning: we might be masking a severe error under a harmless boolean. the throwing nature of this function was intended to allow us to have a "ternary state". Keys might be present or not, and that's fine. But then, it can be the case that they could also be not present or unavailable for other reasons (errors or inconsistencies of some sort) and we want to be able to tell that "not present" case apart.

Reply:
Agreed, PR 217 (https://github.com/zcash/secant-ios-wallet/pull/217) brings it back.